src/Interpreters/InterpreterSelectQuery.cpp

```diff
@@ -2299,22 +2299,27 @@ void InterpreterSelectQuery::executeFetchColumns(QueryProcessingStage::Enum proc
         /// need replay operation
```

Comment:
For community code or file, we should add `proton:starts/ends` to contains ourselves changes
```c++
/// proton: starts.
 ...your changes
/// proton: ends.